control_panel_api/tests/test_pagination.py

```diff
@@ -0,0 +1,29 @@
+from model_mommy import mommy
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+from control_panel_api.tests.test_views import AuthenticatedClientMixin
+
+
+class TestPagination(AuthenticatedClientMixin, APITestCase):
+    def test_pagination(self):
+        mommy.make('control_panel_api.App', 120)  # 120 exceeds the 100 default
+
+        app_list_url = reverse('app-list')
+
+        fixtures = (
+            (app_list_url, 100, True),  # default paging no query param
+            (app_list_url + '?page_size=50', 50, True),  # less than default
+            (app_list_url + '?page_size=130', 120, False),  # more than default
+            (app_list_url + '?page_size=0', 120, False),  # return all
```

Comment:
The same goes here, `?page_size={default_page_size - 50}` would make the intent obvious